### DISCUSSION TITLE
这种行为是不是违法了(

### DISCUSSION THREAD

**Discussion by Chormore-EDGE** [Meloong-Git/PCL#1975] [提问] [answered]
如2b贴吧https://tieba.baidu.com/p/6877222401

**Comment by Pigeon0v0** [comment, marked as answer]
你可以加群然后获取他那份内测版，之后将内测版样本提交给龙猫

**Reply by Chormore-EDGE** [reply]
搜不到了，估计不是隐藏就是解散了

**Reply by iPodtouch6557** [reply]
> 搜不到了，估计不是隐藏就是解散了

真的找不到了

**Comment by iPodtouch6557** [comment]
我刚注意到日期，都2020年的了……不管了吧，大概率是不会有回音了

**Reply by Chormore-EDGE** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/114737601/3b6782da-c64d-4918-92a0-9ad1cf25176b)
看日期，1年过去了还回复)

**Comment by Chormore-EDGE** [comment]
至少2021年这个人还是在的
